engine/engine/content/builtins/manifests/android/AndroidManifest.xml

```diff
@@ -22,7 +22,7 @@
 
         <activity android:name="com.dynamo.android.DefoldActivity"
                 android:label="{{project.title}}"
-                android:configChanges="orientation|screenSize|keyboardHidden"
+                android:configChanges="fontScale|keyboard|keyboardHidden|locale|mcc|mnc|navigation|orientation|screenLayout|screenSize|smallestScreenSize|touchscreen|uiMode"
```

Comment:
There's a two more here:

https://developer.android.com/guide/topics/manifest/activity-element#config

* density - API Level 24
* layoutDirection - API Level 17

Both are above our minimum API level of 16. Is that why we didn't add them? What happens if either of these change?

Reply:
Yes, I don't want to increase minimum API because of that. 
If one of these events happen then OS call
APP_CMD_PAUSE
APP_CMD_TERM_WINDOW
APP_CMD_STOP
APP_CMD_SAVE_STATE
APP_CMD_DESTROY

More likely for the end-user it will be like a black screen or app closing.

These events are pretty specific. One for devices with a few displays another one for switching RTL <-> LTR.
I believe we cover 98-99% of cases by adding all events except these two.

Reply:
Makes sense! It's definitely an improvement!